README.md

```diff
@@ -270,6 +270,21 @@ set by the [Proselint tool](http://proselint.com/). Since GatorGrader can run an
 arbitrary command and check its error code, it is also possible to integrate it
 with a wide variety of other linters, code formatters, and testing tools.
 
+### Issues
+
+We have seen many cases of the `gradle grade` command unexpectedly
+not working correctly becuase GatorGrader could not be pulled correctly.
+The main reason for this issue is out of date dependency versions not
+lining up with the current release of GatorGrader. However, there
+have beeen some instances of this issue without a clear explaination.
+To address this issue, please locate the `virtualenvs` and `gatorgrader`
+directories through the `.local/share` path. Delete `virtualenvs` and
```

Comment:
You may also want to mention that in some cases these directories are in the DockaGator cache volume (as used in the [DockaGator README](https://github.com/GatorEducator/dockagator/blob/master/README.md) examples), so running `docker volume rm dockagator` will clear the docker volume which contains these virtualenvs and GatorGrader installation, if that is how they were created.

Reply:
Thanks for your feedback @Michionlion! I have updated the PH accordingly, please let me know if there are any further changes to be made.

Reply:
No problem, @schultzh! I've added a few more issues that I didn't catch the first time in a new review.